### ISSUE TITLE
import problem

### ISSUE THREAD

**Issue by bruna-paz** [cap1tan/wafermap#1]
* Wafermap version: (2021.10.8)
* Python version: 3.8.13
* Operating System: Windows 10

### Description

Can't import.

### What I Did

```
pip install wafermap
Requirement already satisfied: wafermap in c:\users\yami_\anaconda3\lib\site-packages (0.2.5)
Requirement already satisfied: branca<0.7.0,>=0.6.0 in c:\users\yami_\anaconda3\lib\site-packages (from wafermap) (0.6.0)
Requirement already satisfied: folium<0.15.0,>=0.14.0 in c:\users\yami_\anaconda3\lib\site-packages (from wafermap) (0.14.0)
Requirement already satisfied: Pillow<10.0.0,>=9.5.0 in c:\users\yami_\anaconda3\lib\site-packages (from wafermap) (9.5.0)
Requirement already satisfied: numpy<2.0.0,>=1.24.2 in c:\users\yami_\anaconda3\lib\site-packages (from wafermap) (1.24.4)
Requirement already satisfied: html2image<3.0.0,>=2.0.3 in c:\users\yami_\anaconda3\lib\site-packages (from wafermap) (2.0.3)
Requirement already satisfied: jinja2 in c:\users\yami_\anaconda3\lib\site-packages (from branca<0.7.0,>=0.6.0->wafermap) (2.11.2)
Requirement already satisfied: requests in c:\users\yami_\anaconda3\lib\site-packages (from folium<0.15.0,>=0.14.0->wafermap) (2.27.1)
Requirement already satisfied: MarkupSafe>=0.23 in c:\users\yami_\anaconda3\lib\site-packages (from jinja2->branca<0.7.0,>=0.6.0->wafermap) (2.0.1)
Requirement already satisfied: urllib3<1.27,>=1.21.1 in c:\users\yami_\anaconda3\lib\site-packages (from requests->folium<0.15.0,>=0.14.0->wafermap) (1.26.9)
Requirement already satisfied: idna<4,>=2.5 in c:\users\yami_\anaconda3\lib\site-packages (from requests->folium<0.15.0,>=0.14.0->wafermap) (3.3)
Requirement already satisfied: charset-normalizer~=2.0.0 in c:\users\yami_\anaconda3\lib\site-packages (from requests->folium<0.15.0,>=0.14.0->wafermap) (2.0.4)
Requirement already satisfied: certifi>=2017.4.17 in c:\users\yami_\anaconda3\lib\site-packages (from requests->folium<0.15.0,>=0.14.0->wafermap) (2021.10.8)

import wafermap
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "C:\Users\yami_\Anaconda3\lib\site-packages\wafermap\__init__.py", line 5, in <module>
    from wafermap.wafermap import WaferMap
  File "C:\Users\yami_\Anaconda3\lib\site-packages\wafermap\wafermap.py", line 25, in <module>
    class WaferMap:
  File "C:\Users\yami_\Anaconda3\lib\site-packages\wafermap\wafermap.py", line 45, in WaferMap
    cell_margin: tuple[float, float] = (0.0, 0.0),
TypeError: 'type' object is not subscriptable
```

**Comment by MatthieuDartiailh** [comment]
Upgrading to Python 3.10+ would fix that particular issue. 

@cap1tan are you interested in supporting Python 3.9 ? If yes I could make a PR, otherwise it would make sense to update the supported Python versions.

**Comment by cap1tan** [comment]
> Upgrading to Python 3.10+ would fix that particular issue.
> 
> @cap1tan are you interested in supporting Python 3.9 ? If yes I could make a PR, otherwise it would make sense to update the supported Python versions.

Thanks for flagging. Should be fixed now in new v0.2.6.
